API/src/models/Users.js

```diff
@@ -23,6 +23,18 @@ module.exports = (sequelize) => {
             msg: 'El apellido debe tener entre 1 y 30 caracteres'
         }
     },
+    phone:{
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        unique:true,
+        validate:{
+            isPhone(value){
```

Comment:
no podes usar is aca? , tipo para respetar la estructura 
![image](https://github.com/EcommerceProyect/BookStore-E-commerce/assets/103128953/901d8e4e-f412-42a5-9562-c0584ae1d6be)

Reply:
Hecho